app/models/import.rb

```diff
@@ -0,0 +1,59 @@
+require 'csv'
+
+class Import < ActiveRecord::Base
+
+  def initialize(params = {})
+    file = params.delete(:file)
+    super
+    if file
+      self.filename = sanitize_filename(file.original_filename)
+      self.content_type = file.content_type
+      self.file_contents = file.read
```

Comment:
What would happen if the `file` would have about 100_000 rows?
How would you deal with it?